rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/90_sig_text.yml

```diff
@@ -1,151 +0,0 @@
----
```

Comment:
I renamed this file, replace `index` with `_bulk`, and added a ton more tests. It doesn't seem to detect the rename because its barely the same file any more....